### PR TITLE
Cleanup unused frontend code

### PR DESCRIPTION
**Pull request by MarcoDaniels**
Cleanup code:

- Removed unnecessary jQuery files from `index.html`
- Run [elm-review-unused](https://github.com/jfmengels/node-elm-review#try-it-out) to cleanup unused Elm code.

`elm-test` is not really working as `frontend/tests/Example.elm` is just the example and it seems to be some issue with test-dependencies, will probably tackle that in another PR.

### CODE REVIEW

**Comment by MarcoDaniels** [comment]
There's seems to be a little inconsistency on what package manager to use for the frontend. The README recommends using `yarn` to run the project, but the frontend contains a `package-lock.json` generated by `npm`.

Should the references to yarn be replaced with npm?

File: frontend/src/index.html [hunk 1]
```diff
@@ -2,29 +2,24 @@
...
 <head>
-  <meta charset="utf-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>NixOS Search</title>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>NixOS Search</title>
 
-  <script type="text/javascript" src="https://nixos.org/js/jquery.min.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
```
Comment (ncfavier) [inline frontend/src/index.html:9]: Can we remove the duplicate `<meta name=viewport`?

**Review by ncfavier — APPROVED** [review]

**Comment by ncfavier** [comment]
> Should the references to yarn be replaced with npm?

Not sure, cc @garbas

File: frontend/src/index.html (hunk 1 above)
Comment (MarcoDaniels) [reply frontend/src/index.html]: Yes, missed that one.

**Comment by garbas** [comment]
> > Should the references to yarn be replaced with npm?
> 
> Not sure, cc @garbas

I think this was there to deal with the transition from old search implementation. I'd say remove but keep listening if somebody complains.

also regarding yarn. i think we were using it with yarn2nix and we can safely switch to npm now since we are using npmlock2nix.

**Pull request merged**